bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BRefValueArray.java

```diff
@@ -76,7 +77,12 @@ public BValue copy() {
     
     @Override
     public String stringValue() {
-        StringJoiner sj = new StringJoiner(", ", "[", "]");
+        StringJoiner sj;
+        if (arrayType instanceof BTupleType) {
```

Comment:
Can use `arrayType.getTag() == TypeTags.TUPLE_TAG` instead of `instanceof`

Reply:
5dd1c24c9776374598f6707d767294dd864598ad